server/services/deployment/debian.ts

```diff
@@ -0,0 +1,47 @@
+/*
+ Service to collect version info from Debian
+*/
+
+import httpget from "../../util/httpget";
+import DataService from "../data-service";
+
+// https://sources.debian.org/api/src/keyman-config/
+const HOST = 'sources.debian.org';
+const PATH_PREFIX = '/api/src/';
+
+const service = {
+	get: function (tier: string, suite: string) {
+		switch (tier) {
+			case 'alpha':
+			// case 'beta':
+				return null;
+			default:
+				{
+					const component: string = tier == 'stable' ? 'keyman-config' : 'keyman';
```

Comment:
Currently yes - stable still has Keyman 14, beta will have Keyman 15 which has the consolidated source package. But then - since it's the same package repo as soon as the beta package gets accepted in Debian it'll also change the name for stable. But what happens then if we query for keyman-config? I don't know. Probably we'll get an error then.

Seems like we'll have to think how we deal with beta versions and Debian. Do we publish beta versions to Debian? If we publish them to `unstable` (`sid`) (which is kind of Debian staging area) they will replace the stable version. On the other hand that is the easiest way to get them into the next Debian/Ubuntu release since usually the won't do major version upgrades of a package after code freeze.

For now I think we should leave it with the two names and then fix it as things change.

Reply:
> For now I think we should leave it with the two names and then fix it as things change.

OK, but `keyman` gives a 404 in the query.

Reply:
yes, because it's hasn't been accepted yet.